wild/tests/sources/duplicate_strong_symbols.c

```diff
@@ -0,0 +1,14 @@
+//#AbstractConfig:default
+//#Config:executable:default
+//#SkipLinker:ld
+//#CompArgs:-fcommon
+//#Object:duplicate_strong_symbols0.c
+//#Object:duplicate_strong_symbols1.c
+//#Object:exit.c
+//#LinkerDriver:none
+//#LinkArgs:--shared
```

Comment:
It seems fine to build either a shared object (like you are now) or an executable. If you're building a shared object, then you don't need a `_start` function.